centreon/www/install/sql/centreon/Update-DB-23.04.5.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `topology` WHERE `topology_name` = 'About';
+INSERT INTO `topology` (`topology_name`, `topology_url`, `readonly`, `is_react`, `topology_parent`, `topology_page`, `topology_order`, `topology_group`) VALUES ('About', '/administration/about', '1', '1', 5,506,15,1);
```

Comment:
You should `UPDATE` the entry instead of `DELETE / INSERT`
Otherwise, some relations with foreign key could be deleted (ie acl topology relations)